bigquery_etl/query_scheduling/dags.py

```diff
@@ -0,0 +1,70 @@
+"""Represents a collection of configured Airflow DAGs."""
+
+from itertools import groupby
+import yaml
+
+from bigquery_etl.query_scheduling.dag import Dag
+
+
+class Dags:
```

Comment:
I'd prefer to call this something a bit more verbose, like `DagList` since it's difficult to tell the difference between `Dag` and `Dags` at a glance.